zxcvbn.opam

```diff
@@ -1,6 +1,6 @@
 opam-version: "2.0"
-maintainer: "Nathan Rebours <nathan@cryptosense.com>"
-authors: "Nathan Rebours <nathan@cryptosense.com>"
+maintainer: "James Owen <james@cryptosense.com>"
+authors: "James Owen <james@cryptosense.com>"
```

Comment:
Wouldn't it make sense to use a Cryptosense contact here?  Almost all our packages have obsolete information at the moment.

Reply:
Authors is a multi field so you can add new authors without necessarily removing the original one :stuck_out_tongue:

Reply:
@bbc2 Yeah we should probably set up an email for this that just gets forwarded to the whole team.

@NathanReb: Didn't realise that - I will change it back, but would you like it swapped to another email? Out of interest, in the community do people ever contact the author (meaning it needs to be a current address) or not?

Reply:
Yeah you can use `nathan.p.rebours@gmail.com` although I doubt people ever contact authors, especially when they differ from the current maintainer!